src/main/java/io/candydoc/plugin/model/DependencyDto.java

```diff
@@ -2,12 +2,14 @@
 
 import lombok.Builder;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.extern.jackson.Jacksonized;
 
 @Jacksonized
 @Builder
 @Getter
+@Setter
 public final class DependencyDto {
   private final String refersTo;
-  private boolean allowed;
+  private DependencyAuthorization authorization;
```

Comment:
If we remove the deterred dependencies from command and event, deterred dependencies does not exist at all. Moreover, does a deterred dependency and a forbidden dependency are the same thing ? Currently, a forbidden dependency is here to tell the user for dependency issues, without breaking its code.